getting-started/templates/pilot-sizing.yaml

```diff
@@ -0,0 +1,891 @@
+# Default values for SystemLink pilot resources and replicas.
```

Comment:
Do we want some documentation as to what this is and why someone would use it?

Do we want internal-facing documentation on how we arrived at these numbers?

Reply:
I updated the comments at the top of the file to better describe what this file does and how to use it.

Reply:
Discussed with Mark.  I will upload the Jupyter notebooks I used for generating the data in order to test the UI experience at this small scale.